Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.12.2-alpine3.19
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHOUNNBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
```

Comment:
Consider using `COPY requirements.txt ./` to ensure the file is copied to the current working directory set by `WORKDIR`.